systemtest/src/test/java/io/strimzi/systemtest/tracing/TracingST.java

```diff
@@ -556,7 +551,7 @@ void testProducerConsumerMirrorMakerService() {
         KafkaClientsResource.consumerWithTracing(KafkaResources.plainBootstrapAddress(kafkaClusterTargetName)).done();
 
         KafkaMirrorMakerResource.kafkaMirrorMaker(CLUSTER_NAME, kafkaClusterSourceName, kafkaClusterTargetName,
-            ClientUtils.generateRandomConsumerGroup(), 1, false)
+                "my-group" + new Random().nextInt(Integer.MAX_VALUE), 1, false)
```

Comment:
Why this change is needed?

Reply:
Good catch! I think I little bit messed up the rebase